schema/mysql/v57/cadence/schema.sql

```diff
@@ -0,0 +1,331 @@
+CREATE TABLE domains(
+/* domain */
+  id CHAR(36) PRIMARY KEY NOT NULL,
+  name VARCHAR(255) UNIQUE NOT NULL,
+  status INT NOT NULL,
+  description VARCHAR(255) NOT NULL,
+  owner_email VARCHAR(255) NOT NULL,
+  data BLOB,
+/* end domain */
+  retention INT NOT NULL,
+  emit_metric TINYINT(1) NOT NULL,
+/* end domain_config */
+  config_version BIGINT NOT NULL,
+  notification_version BIGINT NOT NULL,
+  failover_notification_version BIGINT NOT NULL,
+  failover_version BIGINT NOT NULL,
+  is_global_domain TINYINT(1) NOT NULL,
+/* domain_replication_config */
+  active_cluster_name VARCHAR(255) NOT NULL,
+  clusters BLOB
+/* end domain_replication_config */
+) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+
+CREATE TABLE domain_metadata (
+  notification_version BIGINT NOT NULL
+);
+
+INSERT INTO domain_metadata (notification_version) VALUES (0);
+
+CREATE TABLE shards (
+	shard_id INT NOT NULL,
+	owner VARCHAR(255) NOT NULL,
+	range_id BIGINT NOT NULL,
+	stolen_since_renew INT NOT NULL,
+	updated_at TIMESTAMP(3) NOT NULL,
+	replication_ack_level BIGINT NOT NULL,
+	transfer_ack_level BIGINT NOT NULL,
+	timer_ack_level TIMESTAMP(3) NOT NULL DEFAULT '1970-01-01 00:00:01.000',
```

Comment:
Change all TIMESTAMP to DATETIME(6) with exception of updated_at column. This is to avoid autoupdating them. See https://dev.mysql.com/doc/refman/5.5/en/timestamp-initialization.html